README.md

```diff
@@ -45,6 +45,21 @@ for configuring / running the UI server.
 
 ## Development
 
+####System Dependencies 
+
+The server code is written in python3. Confirm your version using command:
+
+    python -V
+
+If Python is missing or an earlier version, install python3 and pip3. 
+
+PyYaml is required for running tests. Install it using pip3:
+
+    pip3 install pyyaml
+
+
+####Deploying to Localhost
```

Comment:
Need a space -> #### Deploying to Localhost

Reply:
Added spaces to both subsection headers